MasomodeDLC.csproj

```diff
@@ -30,7 +30,4 @@
       <HintPath>..\..\references\mods\ThoriumMod.dll</HintPath>
     </Reference>
   </ItemGroup>
-  <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
```

Comment:
Not sure why this is removed.

Reply:
Couldn't build with that pre-build task in the project file, but it worked as soon as I removed it. Likely because I have my Terraria assets in a different location than the default. From my experience removing this would make it universally buildable, as opposed to only being able to build it if your Terraria is installed into that exact path.

Reply:
You could just change the path. Have mine on D drive.

Reply:
Undid this commit.